plotly/offline/offline.py

```diff
@@ -236,13 +247,39 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
 
     display(HTML(plot_html))
 
+    if image:
+        warnings.warn('For higher resolution images and more export '
+                      'options, consider making requests to the '
+                      'Plotly image servers. Type help(\'py.image\') '
+                      'for more details.', UserWarning)
```

Comment:
🐄 I don't think you should use a warning here. It's a good idea in theory, but in notebooks, it often makes it look like we have broken code or something.